nuxeo-vision-core/src/main/java/org/nuxeo/vision/core/google/GoogleVisionDescriptor.java

```diff
@@ -30,9 +30,9 @@
 @XObject("configuration")
 public class GoogleVisionDescriptor {
 
-    public static final String CRED_ENV_VARIABLE = "NUXEO_GOOGLE_APPLICATION_CREDENTIALS";
+    public static final String CRED_ENV_VARIABLE = "org.nuxeo.vision.test.credential.file";
```

Comment:
You can call this `CRED_PROP` and `KEY_PROP` as these are system properties now.